src/aleph/http/core.clj

```diff
@@ -501,13 +515,21 @@
     ;; with transfer encoding chunked
     (if-let [s @stream]
       (do
-        ;; flag that body was not completed succesfully
+        ;; flag that connection was terminated early
```

Comment:
The previous comment `not completed succesfully` was more accurate, no?

Reply:
Well, in this particular use, yes, that's the more accurate cause, but I changed the comment to reflect what/how `complete` is used to communicate downstream. Nothing uses `complete` to learn anything about the body, but to know whether to destroy the connection.

Ugh. Gotta rename this